flowgate-common/src/main/java/com/vmware/flowgate/common/model/ValueUnit.java

```diff
@@ -57,34 +57,16 @@ public void setTime(long time) {
       this.time = time;
    }
 
-   public ValueType getKey() {
+   public String getKey() {
       return key;
    }
 
-   public void setKey(ValueType key) {
+   public void setKey(String key) {
       this.key = key;
    }
 
-   public enum ValueType {
-      TEMP, HUMIDITY, PDU_RealtimeVoltage, PDU_RealtimeVoltagePercent, PDU_RealtimePower,
-      PDU_RealtimePowerPercent, PDU_RealtimeLoad, PDU_RealtimeLoadPercent,AirFlow,AirPressure,
-      ContactClosure,Smoke,Water,Vibration
-   }
-
    public enum MetricUnit {
-       MV(ValueType.PDU_RealtimeVoltage), V(ValueType.PDU_RealtimeVoltage), KV(ValueType.PDU_RealtimeVoltage), VOLTS(ValueType.PDU_RealtimeVoltage),
```

Comment:
The value of the MetricUnit represent the same group actually,   we need to figure out how can we know W, KW are the same kind of Unit.

Reply:
> The value of the MetricUnit represent the same group actually, we need to figure out how can we know W, KW are the same kind of Unit.

Done